about_regular_expressions.exs

```diff
@@ -0,0 +1,36 @@
+defmodule AboutRegularExpressions do
+  use Koans
+
+  think "A simple regular expression" do
+    regex = Regex.compile!("This [W|w]orks!")
```

Comment:
gotta be honest, idk what sigils are. i just though `~r{}` was "the regex thing". so maybe that's proof positive that it doesn't have to scare them off?